app/components/Channels/messages.js

```diff
@@ -43,5 +43,10 @@ export default defineMessages({
     'The amount to be paid in fees for the current set of commitment transactions.',
   base_fee_label: "Node's Fee Base",
   base_fee_description:
-    'The required number of {currencyName} per kilo-weight that the requester will pay at all times, for both the funding transaction and commitment transaction.'
+    'The required number of {currencyName} per kilo-weight that the requester will pay at all times, for both the funding transaction and commitment transaction.',
```

Comment:
New translation strings need to be extracted with `yarn extract-messages`

Reply:
This is wip